interceptor/store.go

```diff
@@ -3,26 +3,13 @@ package interceptor
 import (
 	"time"
 
+	"github.com/breez/lspd/shared"
```

Comment:
I know that it's a nit about naming and it's already in the branch, but I don't think that the name common or lncommon convey more the meaning than "shared"

Reply:
I'll rename that in a separate PR